<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-26T20:47:46 IST" name="Suite" finished-at="2023-11-26T20:48:01 IST" duration-ms="14114">
    <groups>
    </groups>
    <test started-at="2023-11-26T20:47:46 IST" name="Test" finished-at="2023-11-26T20:48:01 IST" duration-ms="14114">
      <class name="testcases.TC01_Login_OrangeHRM">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:testcases.TC01_Login_OrangeHRM@795509d9]" started-at="2023-11-26T20:47:46 IST" name="setup" finished-at="2023-11-26T20:47:53 IST" duration-ms="6400" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="test_login_OrangeHRM()[pri:0, instance:testcases.TC01_Login_OrangeHRM@795509d9]" started-at="2023-11-26T20:47:53 IST" name="test_login_OrangeHRM" finished-at="2023-11-26T20:48:01 IST" duration-ms="7650" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [There is already an account with this email address. If you are sure that it is your email address, click here to get your password and access your account.] but found [Thank you for registering with Main Website Store.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [There is already an account with this email address. If you are sure that it is your email address, click here to get your password and access your account.] but found [Thank you for registering with Main Website Store.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at testcases.TC01_Login_OrangeHRM.test_login_OrangeHRM(TC01_Login_OrangeHRM.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_login_OrangeHRM -->
        <test-method is-config="true" signature="testDown()[pri:0, instance:testcases.TC01_Login_OrangeHRM@795509d9]" started-at="2023-11-26T20:48:01 IST" name="testDown" finished-at="2023-11-26T20:48:01 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDown -->
      </class> <!-- testcases.TC01_Login_OrangeHRM -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
